### PR TITLE
#BE 3. Create model class#4.1 DBRecordModel with constructor and gett…

### PR DESCRIPTION
**Pull request by Aspink**
…ers created.

#BE 3. Create model class#4.1 DBRecordModel with constructor and getters created.

### CODE REVIEW

File: backend/java/main/DBRecordModel.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package main;
+public class DBRecordModel {
```
Comment (danielvegas) [inline backend/java/main/DBRecordModel.java]: DBRecordModel doesn't say what is the model.
If you create User class, you don't call it DBUserModel :) 
Just saying...

File: backend/java/main/DBRecordModel.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    private final String nickname, city, zipCode, country;
```
Comment (danielvegas) [inline backend/java/main/DBRecordModel.java]: Have you checked what fields are necessary to get the data from Google Maps API ?

File: backend/java/main/DBRecordModel.java (hunk 1 above)
Comment (Aspink) [reply backend/java/main/DBRecordModel.java]: Renamed to User.

File: backend/java/main/DBRecordModel.java (hunk 2 above)
Comment (Aspink) [reply backend/java/main/DBRecordModel.java]: image

So we probably won't use Google Maps. We haven't decided what we're going to use yet, and the given data seems to make sense. We will correct it if necessary.

**Review by MateuszLesisz — APPROVED** [review]

**Pull request merged**